ios/RCTBraintree/RCTBraintree.m

```diff
@@ -121,8 +121,6 @@ - (void)handlePayPalResult:(BTPayPalAccountNonce * _Nullable)tokenizedPayPalAcco
         }
     } else if ( error != nil ) {
         args = @[error.description, [NSNull null]];
-    } else { // per braintree docs, if both error and token are nil, user cancelled
-        args = @[@"USER_CANCELLATION", [NSNull null]];
```

Comment:
is it ok to remove this? did you test what happens when you just cancel without depositing?

Reply:
Nope, i didn't catch that got changed. I think some stuff got messed up between pulling from master and such. Will fix it